lumicks/pylake/kymotracker/kymotracker.py

```diff
@@ -295,15 +295,16 @@ def refine_lines_gaussian(
     overlap_strategy : {'ignore', 'skip'}
         How to deal with frames in which the fitting window of two `KymoLine`'s overlap.
 
-        - 'ignore' : do nothing, fit the frame as-is.
+        - 'fit_multiple' : fit the peaks simultaneously.
```

Comment:
I'm not sure about `"fit_multiple"`, for arguments that users are required to supply I prefer to keep to short names and if possible without special characters (`_`) .... maybe it's just a weird pet peeve of mine. 

At the same time I can't really come up with something better. First thing that comes to mind is `"overlap"` but I don't know if that's descriptive enough. Maybe just `"multiple"`?

Reply:
`multiple` it is!